pkgs/applications/audio/lpd8editor/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, qt5
+, stdenv
+, git
+, fetchFromGitHub
+, cmake
+, alsa-lib
+, qttools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lpd8editor";
+  version = "0.0.16";
+
+  src = fetchFromGitHub {
+    owner = "charlesfleche";
+    repo = "lpd8editor";
+    rev = "v${version}";
+    hash = "sha256-lRp2RhNiIf1VrryfKqYFSbKG3pktw3M7B49fXVoj+C8=";
+  };
+
+  buildInputs = [ qttools ];
+
+  nativeBuildInputs = [
+    cmake
+    alsa-lib
```

Comment:
`alsa-lib` should be in `buildInputs` (you can also enable `strictDeps = true` to check for this)